northstar/src/seccomp/types.rs

```diff
@@ -23,6 +23,14 @@ pub struct Seccomp {
     pub allow: Option<HashMap<NonNullString, SyscallRule>>,
 }
 
+/// SELinux configuration
+#[derive(Clone, Eq, PartialEq, Debug, Serialize, Deserialize, JsonSchema)]
+pub struct Selinux {
+    /// Explicit list of allowed syscalls
+    #[serde(rename = "type")]
+    pub context_type: NonNullString,
```

Comment:
Can we remove the `rename`? Either use `context_type` in them yaml or `r#type` as field name. I don't know what's the better choice.

Reply:
context_type it is.